docker/hive/hive/hive-site.xml

```diff
@@ -45,7 +45,7 @@
 
   <property>
     <name>hive.metastore.warehouse.dir</name>
-    <value>/tmp/test</value>
+    <value>file:///warehouse</value>
     <description>location of default database for the warehouse</description>
   </property>
 
```

Comment:
I have not had success overriding this at the command line.  Might it make sense just to delete this from the configuration file entirely so that we can be certain about being able to (easily) configure it dynamically?

Reply:
Give it a shot. The worst case scenario is it choosing the default